pkgs/development/python-modules/codefind/default.nix

```diff
@@ -0,0 +1,28 @@
+{
+  lib,
+  buildPythonPackage,
+  fetchPypi,
+  hatchling,
+}:
+
+buildPythonPackage rec {
+  pname = "codefind";
+  version = "0.1.7";
+  format = "pyproject";
+
+  src = fetchPypi {
+    inherit pname version;
+    hash = "sha256-ouyKLAGAOZ6oON/NzDRMqJ+XuKopO8F7IrLAI6ugb7w=";
+  };
```

Comment:
fetchFromGitHub or GitLab or wherever is preferred

Reply:
Reference?

Reply:
@veprbl https://github.com/NixOS/nixpkgs/pull/371840#discussion_r1906104135

Reply:
Well, that is not an explanation. Not using fetchFromGithub where possible is just fine.

Reply:
@veprbl Pypi tends to leave out package tests and the like.